### PR TITLE
Milestone 6: Accessibility Checks and Updates

### PR DESCRIPTION
**Pull request by carlylechia**
This is what my partner and I did on this branch:
 1. Did ALL the checks required in the project requirements as seen below:
        - Page titles: one issue found. https://github.com/carlylechia/MY-PORTFOLIO/pull/8/files#:~:text=%3Ctitle%3E-,Chia,-Carlyle%27s%20Portfolio%20Site
        - Image text alternatives** (alt text): three issues found. https://github.com/carlylechia/MY-PORTFOLIO/pull/8/files#:~:text=png%22%20alt%3D%22github-,%2D,-logo%22%3E%3C/a%3E%3C/li
        [
](https://github.com/carlylechia/MY-PORTFOLIO/pull/8/files#:~:text=Vector.png%22%20alt%3D%22-,angellist,-%2Dlogo%22%3E%3C/a%3E%3C/li) 
https://github.com/carlylechia/MY-PORTFOLIO/pull/8/files#:~:text=icon.png%22%20alt%3D%22-,moment,-%2Dicon%22%3E%3C/a%3E%3C/li
       - Text headings: No issues found
        - Color contrast: one issue found. https://github.com/carlylechia/MY-PORTFOLIO/pull/8/files#:~:text=gradient(180.45deg%2C%20rgba(-,15,-%2C%2015%2C%2015%2C%200.74
        - Resize: issues found. https://github.com/carlylechia/MY-PORTFOLIO/pull/8/files#:~:text=height%3A-,70vh,-%3B   
        - Interaction: No issues were found
        - Moving content: No issues were found.
        - Multimedia: No issues were found.
        - The basic structure of the page: No issues were found.
 2. Updated my code to make it more accessible where there was the need to do so.

### CODE REVIEW

**Comment by ShoiraTa** [comment]
Hi team,

:thumbsup: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.

There are some issues that you still need to work on to go to the next project but you are almost there!



 ## Required Changes  :recycle: 

- [ ] Based on project requirements, you have to list all points that you have checked:
```
- Page titles
- Image text alternatives** (<img alt="alt text">)
- Text headings
- Color contrast
- Resize
- Interaction
- Moving content
- Multimedia
- The basic structure of the page
```
Please all all these points and make sure that each point on the list included the result of your evaluation. 
> If everything was ok, make a statement about that (e.g. "Page titles - no issues found").
> If you needed to change something, link to the corresponding line of code in your pull request changes.
***
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

* * *
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -16,7 +16,7 @@
     <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins:wght@300&family=Roboto:wght@100;400&display=swap" rel="stylesheet">
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
     <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@300&family=Poppins:wght@300&family=Roboto:wght@100;400&display=swap" rel="stylesheet">
-    <title>Carlyle's-Portfolio-site</title>
+    <title>Chia Carlyle's Portfolio Site</title>
```
Comment (elmar8287) [inline index.html:19]: - [ ] If you click on line 19 or 20, or any other line, you can see the URL changing. For example, by clicking line 19 we can see the URL https://github.com/carlylechia/MY-PORTFOLIO/pull/8/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R19. You can copy this link for your report, for your the first point:

- Page titles - one issue found [link](https://github.com/eng-mohamed-eldeeb/protofolio/pull/5/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051L22)

Please, consider this technic for all found issues.

**Review by elmar8287 — CHANGES_REQUESTED** [review]
Hi @carlylechia, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️

- [ ]  Check the comments under the review.
- [ ]  When an issue is found please link to the corresponding line of code in the Pull Request changes. So it's easy to trace the issue. You can familiar how to do it via [this link](http://www.chasewoodford.com/blog/linking-highlighted-lines-code-github/#:~:text=To%20link%20to%20multiple%20lines,php%23L117%2DL148)

- In the comment under review, I explain how to do it, so you can easily fix it for the next submission.

## Optional suggestions

- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Lordkaito — APPROVED** [review]
Hi @carlylechia, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**